README.md

```diff
@@ -1,3 +1,5 @@
+Cameron's Trello Board URL: https://trello.com/b/cjRCp0Co/lambdamud
```

Comment:
Good that you added a link to your URL here. It's a small thing but many students forget/ignore that instruction.